src/mockServiceWorker.js

```diff
@@ -316,7 +321,7 @@ function sendToClient(client, message) {
       resolve(event.data)
     }
 
-    client.postMessage(message, [channel.port2])
+    client.postMessage(message, [channel.port2].concat(transfer || []))
```

Comment:
@95th, one thing I've changed is added this `.concat()` instead of spreading. I know spreading is quite supported right now but I wanted to make sure we ship worker code that most of the browsers can understand.